scout/utils/ensembl_rest_clients.py

```diff
@@ -29,43 +28,13 @@ def __init__(self, build="37"):
         else:
             self.server = RESTAPI_37
 
-    def ping_server(self):
```

Comment:
☑️ Not needed with download all really. And the whole client is basically obsoleted by shug.